### ISSUE TITLE
`TensorNetwork.replace!` raises `UndefVarError: P not defined` in Tenet v0.7.3

### ISSUE THREAD

**Issue by marcosdiezgarcia** [bsc-quantic/Tenet.jl#190]
I tried to run the following code from the Julia REPL, using **Tenet version 0.7.3**, but raises an error `UndefVarError: 'P' not defined`:
```julia
using Tenet
ketzeros = zeros(Product, 2)
adjoint(Quantum(ketzeros))
```

Stacktrace:
![image](https://github.com/user-attachments/assets/96cda17a-4c98-49cc-8680-4aa86ff2dd13)

However, the above error **does not** occur in **Tenet version 0.7.1**.

I am running Julia version 1.9.3.

**Comment by mofeing** [comment]
Thanks for the report @marcosdiezgarcia! We're now on vacations so we won't be able to properly fix and publish a new version until early September.

Meanwhile, you can get the bug fixed by defining the following in your code:
```julia
```

**Comment by mofeing** [comment]
@marcosdiezgarcia The problem should be solved in v0.7.4
